tmc-langs-util/pom.xml

```diff
@@ -34,6 +41,18 @@
             <version>1.0-SNAPSHOT</version>
             <type>jar</type>
         </dependency>
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
```

Comment:
Never mind, found it! The package name of the import was misleadingly different.